src/util/profile/profile.ts

```diff
@@ -25,14 +25,15 @@ export interface UpdatableProfile {
 }
 
 export interface Profile extends UpdatableProfile {
-  readonly accessToken: Promise<TokenValueType>;
+  readonly accessToken: Promise<string>;
```

Comment:
Is this actually an issue for you? If not, I'm going to digging into the token stuff anyway in the next couple of days (to support --token switch) and can fix this up as part of that work.